Framework/API/inc/MantidAPI/ISpectrum.h

```diff
@@ -1,13 +1,17 @@
 #ifndef MANTID_API_ISPECTRUM_H_
 #define MANTID_API_ISPECTRUM_H_
 
-#include "MantidKernel/System.h"
 #include "MantidGeometry/IDTypes.h"
+#include "MantidHistogramData/Histogram.h"
+#include "MantidKernel/System.h"
 #include "MantidKernel/cow_ptr.h"
 
 #include <set>
 
 namespace Mantid {
+namespace DataObjects {
```

Comment:
Good catch. It is a leftover of a `friend` declaration I had to add a while ago, but managed to remove later.